runtime/v2/shim/util.go

```diff
@@ -69,10 +72,15 @@ func Command(ctx context.Context, runtime, containerdAddress, path string, cmdAr
 	return cmd, nil
 }
 
-// BinaryName returns the shim binary name from the runtime name
+// BinaryName returns the shim binary name from the runtime name,
+// empty string returns means runtime name is invalid
 func BinaryName(runtime string) string {
+	// runtime name should format like elem1.elem2
 	parts := strings.Split(runtime, ".")
-	// TODO: add validation for runtime
+	if len(parts) < 2 {
```

Comment:
are we need to check the `format` like `io.containerd.xx.yy` here?

Reply:
From the logic, runtime name is not limited. But I also curious about whether name need 
specific name prefix.